app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.skills: list = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Instead of directly assigning a new list to `self.skills`, use `self.skills.extend(['JavaScript', 'HTML', 'CSS'])` or `self.skills += ['JavaScript', 'HTML', 'CSS']` to add these skills to the inherited list. This follows the checklist guideline to use `extend()` or `+=` for adding multiple values to a list .